### PR TITLE
Do not use mask layer for hazardset quality calculation

### LINKED ISSUE #618: Data Quality should not be read from mask layers
Flood mask layer have data quality currently set to 0,0.
In the admin hazardsets page, data quality rating of layers with a mask show as 0,0, but in geonode the values are 5 (calculation method) and 2 (scientific).

It seems the admin page is incorrectly showing the data quality from mask layer, not RP layer.

It seems the RP layer data quality values are used in the decision tree, as Alfieri is used correctly for France (If values 0,0 were being used, data source would show as GAR15)

### PR DESCRIPTION
**Pull request by arnaud-morvan**
Fix #618.

To update an existing database, execute : 
```
UPDATE processing.hazardset SET complete = FALSE
```
and ``make complete``.

This will update quality fields in hazardset table.

### CODE REVIEW

File: thinkhazard/processing/completing.py [hunk 1]
```diff
@@ -131,6 +131,7 @@ def complete_hazardset(self, hazardset_id, dry_run=False):
             func.min(Layer.calculation_method_quality),
             func.min(Layer.scientific_quality)) \
             .filter(Layer.hazardset_id == hazardset.id) \
+            .filter(Layer.mask == false()) \
```
Comment (pgiraud) [inline thinkhazard/processing/completing.py]: Why not simply using `Layer.mask.is_(False)` or `Layer.mask.isnot(True)`?

Comment (arnaud-morvan) [inline thinkhazard/processing/completing.py]: I don't mind, no specific reason, you say it's simpler, it seems equal for me.

Comment (pgiraud) [inline thinkhazard/processing/completing.py]: No need to import anything.

**Comment by pgiraud** [comment]
Looks good to me. Please merge.

**Pull request merged**